src/lib/data/github-repo.ts

```diff
@@ -4,6 +4,8 @@ export interface GitHubApiResponse {
 }
 
 export class GitHubRepo {
+  private static readonly MAX_COMMITS_TO_FETCH = 100;
```

Comment:
comment why. and make this a global variable in this file, doesn't belong to `GitHubRepo`